instagram/models.py

```
@@ -295,3 +296,59 @@ class Meta:
+    story_credit = models.ForeignKey(StoryCredit, on_delete=models.CASCADE)
+    payment = models.ForeignKey(Payment, on_delete=models.CASCADE)
+
+    credit = models.IntegerField(default=0)
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        verbose_name = "Story Credit Payment"
+        verbose_name_plural = "Story Credit Payments"
+        ordering = ["-created_at"]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)